TestAssignmentDoItSoftware/TestAssignmentDoItSoftware/MyTaskListScreen/MyTaskListScreenViewController.m

```diff
@@ -0,0 +1,248 @@
+    
+    [self.navigationController pushViewController:remindersScreen animated:YES];
+    
+}
+
+
+-(void)goToSortScreen{
+    
+    SortOrderForTasksViewController *sortOrderForTasks = [[SortOrderForTasksViewController alloc] init];
+    
+    [self.navigationController pushViewController:sortOrderForTasks animated:YES];
+    
+}
+
+
+-(void)goAddTaskScreen{
+    
+    NSLog(@"goAddTaskScreen");
+
+    EditTaskScreenViewController *editTaskScreen = [[EditTaskScreenViewController alloc] init];
+    
+    NSMutableDictionary *dictionaryTask = [[NSMutableDictionary alloc] init];
+    [editTaskScreen setDictionaryTask:dictionaryTask];
+
+    [self.navigationController pushViewController:editTaskScreen animated:YES];
+    
+}
+
+
+-(void)doRefresh{
+    
+    NSLog(@"Do refresh");
+    
+    TASingleton *singleton = [[TASingleton alloc] init];
+    arrTasks = [singleton getTasks];
+    
+    [tblTasks reloadData];
+
+    [self stopTableRefresh];
+    
+}
+
+
+-(void)stopTableRefresh{
+    
+    [refreshControl endRefreshing];
+    
+}
+
+
+-(void)viewWillAppear:(BOOL)animated{
+    
+    [super viewWillAppear:animated];
+    
+    [self doRefresh];
+    
+    [UIApplication sharedApplication].applicationIconBadgeNumber = 0;
+    
+    UNUserNotificationCenter *localNotificationCenter = [UNUserNotificationCenter currentNotificationCenter];
+    
+    [localNotificationCenter getNotificationSettingsWithCompletionHandler:^(UNNotificationSettings * _Nonnull settings) {
+        
+        if (settings.authorizationStatus != UNAuthorizationStatusAuthorized) {
+            
+            appDelegate.localNotifications = false;
+            
+        }else{
+            
+            appDelegate.localNotifications = true;
+            
+        }
+        
+    }];
+
+    NSLog(@"token: %@", appDelegate.token);
+    
+}
+
+
+#pragma mark - UI tableview
+
+
+- (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath { 
+    
+    UITableViewCell *cell = [[UITableViewCell alloc] init];
+    
+    cell.textLabel.text = @"text cell";
+
+    MyTaskTableViewCell *myTaskTableViewCell = [tblTasks dequeueReusableCellWithIdentifier:@"MyTaskTableViewCell"];
+    
+    if (myTaskTableViewCell == nil) {
+        
+        NSArray *topLevelObjects = [[NSBundle mainBundle] loadNibNamed:@"MyTaskTableViewCell" owner:self options:nil];
+        myTaskTableViewCell = [topLevelObjects objectAtIndex:0];
+        
+    }
+
+    myTaskTableViewCell.backgroundColor = [UIColor clearColor];
+    
+    if ([myTaskTableViewCell respondsToSelector:@selector(setSeparatorInset:)]) {
+        [myTaskTableViewCell setSeparatorInset:UIEdgeInsetsZero];
+    }
+    
+    if ([myTaskTableViewCell respondsToSelector:@selector(setPreservesSuperviewLayoutMargins:)]) {
+        [myTaskTableViewCell setPreservesSuperviewLayoutMargins:NO];
```

Comment:
Again, it's a style thing, but there's `NO` in one places, `false` in the others. It all should be uniform.

Reply:
@farcaller also another case, there are checks for a new api what available in modern ios. here better to use 
``` 
if (@available(iOS 12.0, *)) { 
myTaskTableViewCell.preservesSuperviewLayoutMargins = NO;
} else {
//some old solution
}
```
but they are old api what available from ios 8+. So this checks don't have any sence.